web/src/pages/Route/List.tsx

```diff
@@ -35,24 +35,28 @@ const Page: React.FC = () => {
     },
     {
       title: formatMessage({ id: 'page.route.domainName' }),
-      dataIndex: 'hosts',
       hideInSearch: true,
-      render: (_, record) =>
-        (record.hosts || []).map((host) => (
-          <Tag key={host} color="geekblue">
-            {host}
+      render: (_, record) => {
+        const list = record.hosts || (record.host && [record.host]) || [];
```

Comment:
just want to confirm: Is it possible to return both hosts and hosts with different values?

Reply:
If hosts exist, then use it, or use host.